grails-app/controllers/au/org/ala/spatial/service/ManageLayersController.groovy

```diff
@@ -30,7 +30,7 @@ import org.json.simple.parser.JSONParser
 
 import java.text.SimpleDateFormat
 
-@RequireAdmin
+@RequirePermission
```

Comment:
As noted in the LoginInterceptor, `@RequirePermission` should be replaced with a test for adminUser OR apiKey.